specification/columns/effectivecost.md

```diff
@@ -1,13 +1,13 @@
 # Effective Cost
 
-Effective Cost represents a cost inclusive of the impacts of all reduced rates and discounts, augmented with the [*amortization*](#glossary:amortization) of relevant purchases (one-time or recurring) paid to cover future eligible charges. The *amortized* portion included should be proportional to the [Pricing Quantity](#pricingquantity) and the time granularity of the data. Effective Cost does not mix or "blend" costs across multiple charges of the same service. This cost is denominated in the [Billing Currency](#billingcurrency). The Effective Cost is commonly utilized to track and analyze spending trends.
+Effective Cost represents a cost inclusive of the impacts of all reduced rates and discounts, augmented with the [*amortization*](#glossary:amortization) of relevant purchases (one-time or recurring) paid to cover future eligible [*charges*](#glossary:charge), with the exception that, in the case of a purchase charge paid to cover future eligible charges, it amounts to 0. The *amortized* portion included should be proportional to the [Pricing Quantity](#pricingquantity) and the time granularity of the data. Effective Cost does not mix or "blend" costs across multiple charges of the same service. This cost is denominated in the [Billing Currency](#billingcurrency). The Effective Cost is commonly utilized to track and analyze spending trends.
```

Comment:
I would consider this more of a clarification than an "exception"

"augmented with the amortization of relevant purchases" to me means that in the case of a purchase of a pre-paid expense (e.g. an upfront Reserved Instance) EffectiveCost would include the negative amortization (which is technically a capitalization of the cost) resulting in the EffectiveCost netting to $0

This is not a blocking comment, just sharing my perspective.

Reply:
@kk09v I just wrote the same thing above and didn't see your comment first haha

Reply:
I agree this might be clearer as a clarification rather than an exception.

I hate to expand the scope of the change, but would it be possible to simplify a little here? I'm getting hung up on the first half of the sentence. While accurate, it's a bit hard for me to grok. I have to read it multiple times and strain a bit. Amortization purists might say this isn't completely accurate, but attempting just in case...

Here's what I want to say in the simplest, most direct way:
```suggestion
Effective Cost represents the [*amortized*](#glossary:amortization) cost of the [*charge*](#glossary:charge) after applying all reduced rates, discounts, and the applicable portion of relevant, prepaid purchases (one-time or recurring) that covered this charge. Since amortization breaks down and spreads the original purchase, Effective Cost is always 0 for prepaid purchase charges that were not covered by another purchase. The *amortized* portion included should be proportional to the [Pricing Quantity](#pricingquantity) and the time granularity of the data. Effective Cost does not mix or "blend" costs across multiple charges of the same service. This cost is denominated in the [Billing Currency](#billingcurrency). The Effective Cost is commonly utilized to track and analyze spending trends.
```

Here's a possible middle ground:
```suggestion
Effective Cost represents the cost of the [*charge*](#glossary:charge) after applying all reduced rates, discounts, and any [*amortized*](#glossary:amortization) charges from relevant purchases (one-time or recurring) paid to cover future eligible changes. Since amortization breaks down and spreads the original purchase, Effective Cost is always 0 for prepaid purchase charges that were not covered by another purchase. The *amortized* portion included should be proportional to the [Pricing Quantity](#pricingquantity) and the time granularity of the data. Effective Cost does not mix or "blend" costs across multiple charges of the same service. This cost is denominated in the [Billing Currency](#billingcurrency). The Effective Cost is commonly utilized to track and analyze spending trends.
```

Reply:
the first suggestion out of two included in the current version of the definition